setuptools/msvc.py

```diff
@@ -1170,14 +1231,16 @@ def _unique_everseen(self, iterable, key=None):
                     seen_add(k)
                     yield element
 
-    def _get_content_dirname(self, path):
+    def _get_content_dirname(self, path, slash=True):
```

Comment:
One should not use boolean parameters to twiddle inner behavior on functions. Instead of passing `slash=False`, just use `_get_content_dirname().rstrip('\\')`.